airflow/providers/amazon/aws/utils/exponential_backoff_retry.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please note that if this util is not under `executors/` then it's not really part of the experimental statement we set for the executor.

I'd suggest either move it under executors/utils or add `_` to function name thus we declare it private.

Reply:
This is a good point. I'll move it under `executors/` for now.